x/subscription/keeper/adjustment.go

```diff
@@ -166,7 +166,8 @@ func (k Keeper) GetAdjustmentFactorProvider(ctx sdk.Context, adjustments []types
 	}
 	providers := []string{}
 	providerUsage := map[string]usage{}
-	for _, adjustment := range adjustments {
+	for _, adj := range adjustments {
```

Comment:
is this correct?
when using range isnt adj a copy?
i dont think it is the actual object from the slice

Reply:
reverted. see https://github.com/lavanet/lava/commit/cadc944ac3ff3648cf98a7751411e897bd5f6cfc